### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by trust-the-log** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#541] [General]
Hello somebody can help me with my st Moritz 54175?
I have fine everything but it doesnt work
Somebody had done it?
I am using a D1 mini
Thanks to everybody

**Comment by visualapproach** [comment]
Hi, you need to be much more specific and tell us what you've done, how you did it, and what you've tried and in what way it doesn't work, if anyone should be able to give any advice.

**Reply by trust-the-log** [reply]
I am using this, ho si now Witch pin i have ti use and where, i connect the calbe of the pump and the display like the PCB V2 but it doesn ‘t work
![Uploading IMG_0110.jpeg…]()

**Reply by trust-the-log** [reply]
![IMG_0111](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/136650000/50552ece-cb58-4ff6-aa21-20062e3c328b)

**Reply by trust-the-log** [reply]
I connect the cables like that and using the 2021 pump

**Reply by trust-the-log** [reply]
![IMG_0110](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/136650000/166f4db8-ff39-4966-b99f-60b4e10a5337)

**Reply by trust-the-log** [reply]
This is my board

**Reply by trust-the-log** [reply]
I found the 5v, G and audio
But RX TX and DATA no

**Reply by trust-the-log** [reply]
I understood the problem!
I had to change the pin settings on the D1 mini and poof everything works perfectly on the 54175 St Moritz pump
 

A thousand thanks!!!
